### DISCUSSION TITLE
allowing users to send testnet coins in exchange for notes

### DISCUSSION THREAD

**Discussion by randoman14** [fedimint/fedimint#6175] [Q&A]
Im trying to create a way for users to send testnet coins in exchange for notes, but I cant get anything to work.

I have gateway-cli and fedimint-cli installed

fedimint-cli --version fedimint-cli 0.4.3

 gateway-cli --version fedimint-gateway-cli 0.4.3

**Comment by dpc** [comment]
I do not understand the context here. Need more details about what are you trying to do. Also, you might get better and more interactive support if you ask around on Discord.

**Reply by randoman14** [reply]
I'm looking to set up a system where I can send testnet coins to a server that exchanges them for notes or tokens. However, I'm struggling with configuring a federation that can manage user creation and generate notes.

**Reply by elsirion** [reply]
It's not fully up-to-date, but you'll likely find useful information on how to set up a federation here: https://github.com/fedimint/fedimint/blob/master/docs/setup-docs.md

**Reply by randoman14** [reply]
Ok thanks @elsirion, ill look into that!
